tests/database_connection/test_mongo_connect.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Test mongodb connection module
+"""
+import pytest
+
+from ghga_service_chassis_lib.mongo_connect import DBConnect
+
+DB_URL = "mongodb://db:27017"
+DB_NAME = "test"
+
```

Comment:
Would be cool to have a mongo mock for testing. E.g. something like this:

https://github.com/ghga-de/sandbox-metadata/blob/dev/tests/fixtures/__init__.py#L13-L29
https://github.com/ghga-de/sandbox-metadata/blob/dev/tests/integration/test_dataset_routes.py#L8

Reply:
Hm OK make sense, I will come up with a custom fixture.